tasks/custom_task/custom_task_test.py

```diff
@@ -0,0 +1,23 @@
+import inspect
+from invoke import Task
+import invoke
+import unittest
+
+expected_content = """    def __call__(self, *args: Any, **kwargs: Any) -> T:
+        # Guard against calling tasks with no context.
+        if not isinstance(args[0], Context):
+            err = "Task expected a Context as its first arg, got {} instead!"
+            # TODO: raise a custom subclass _of_ TypeError instead
+            raise TypeError(err.format(type(args[0])))
+        result = self.body(*args, **kwargs)
+        self.times_called += 1
+        return result
+"""
+
+
+class TestTaskCall(unittest.TestCase):
+    def test_task_call_content(self):
+        self.assertEqual(inspect.getsource(Task.__call__), expected_content)
+
+    def test_invoke_version(self):
+        self.assertEqual(invoke.__version__, '2.2.0')
```

Comment:
Is that really useful? The requirements file is there to indicate the version, if we duplicate it there we need to update the test every time we change the version

Reply:
And I guess the UT itself should assert if your code is still valid when we upgrade module versions

Reply:
Same as above, you're right it's not useful, I'll find another way of doing this !